languages/python/oso/tests/test_polar_roles.py

```diff
@@ -591,10 +578,10 @@ def test_role_implication_without_relationship(init_oso):
     """
     oso.load_str(policy)
     with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
+        oso.validate_config()
 
 
-@pytest.mark.skip("TODO: validation")
+@pytest.mark.skip("Can't really test relationship types")
```

Comment:
same question as above re: validating the presence of potentially valid `parent/2` predicates even if we can't validate the crucial dot lookup that validates the relationship

Reply:
Same as above, without a specializer on the output var instead of an isa on the looked up var there's no real way to know in general what you'll get out. Maybe able to do after we change constraints output some.